.ci/build-kit/docker/Dockerfile

```diff
@@ -1,3 +1,13 @@
 # syntax=docker/dockerfile:1
 ARG BASE_IMAGE_TAG=latest
 FROM ghcr.io/everest/everest-ci/build-kit-base:${BASE_IMAGE_TAG}
+
+# Local update for everest-cmake
+ENV EVEREST_CMAKE_PATH=/usr/lib/cmake/everest-cmake
+# renovate: datasource=github-releases depName=EVerest/everest-cmake versioning=loose
+ENV EVEREST_CMAKE_VERSION=6ac5e03f226b5545d72ca16f3faa4984461e30f6
+RUN rm -rf ${EVEREST_CMAKE_PATH}
+RUN git clone https://github.com/EVerest/everest-cmake.git ${EVEREST_CMAKE_PATH} \
+    && cd ${EVEREST_CMAKE_PATH} \
+    && git checkout ${EVEREST_CMAKE_VERSION} \
+    && rm -r .git
```

Comment:
This should only work once https://github.com/EVerest/everest-core/pull/926 and https://github.com/EVerest/everest-ci/pull/58 are merged